client/my-sites/upgrades/cart/cart-plan-ad.jsx

```diff
@@ -32,12 +33,15 @@ const CartPlanAd = React.createClass( {
 		}
 
 		return (
-			<div className="cart-plan-ad">{
-				this.translate( 'Get this domain for free when you upgrade to {{strong}}WordPress.com Premium{{/strong}}!', {
-					components: { strong: <strong /> }
-				} )
-			} <a href="" onClick={ this.addToCartAndRedirect }>{ this.translate( 'Upgrade Now' ) }</a>
-			</div>
+			<CartAd>
+				{
+					this.translate( 'Get this domain for free when you upgrade to {{strong}}WordPress.com Premium{{/strong}}!', {
```

Comment:
I agree it's better but we would have to load the product list, check that it's loaded... better do it once product-list has been moved to the global state tree.